ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3600,6 +3600,21 @@ AHU_DFSS_DXZC_ECONZ_EFSS2X_HT2ZC_CO2C:
   - EFSS2X
   - HT2ZC
   - CO2C
+  
+AHU_SFVSC_CHWRC_OAFC_SSPC_VOCC_CO2C_EFSM:
+  description: "Single zone AHU with CO2,VOC based outside air control,static pressure based VFD speed control,return air temperature based Chilled water valve modulation."
+  is_canonical: true
+  implements:
+  - AHU
+  - SFVSC
+  - CHWRC
+  - OAFC
+  - SSPC
+  - VOCC
+  - CO2C
```

Comment:
these are zone level controls. which zone? 

i assume this is really return, or an average of zones (or max of zones). if so, need new VOCC and CO2C types to reflect that they are whatever they actually are.